examples/js/scatterplot.js

```diff
@@ -754,4 +754,38 @@ function Scatterplot(el, component, svgClass) {
     return str.replace(' ', '').toLowerCase()
   }
 }
+
+  var scatterplot1Tab1 = document.querySelector('.scatterplot-1-tab-1');
```

Comment:
I would really love to see [this css-only way](https://codepen.io/rm/pen/ldhon) implemented before merging!